### PR TITLE
feat(api/applications): created endpoint for documents (BOAS-458)

### PR DESCRIPTION
**Pull request by Xfarya**
## Describe your changes
created endpoint for documents

## Issue ticket number and link
BOAS-458 https://pins-ds.atlassian.net/browse/BOAS-458

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Comment by mayya-bond-kc** [comment]
@Xfarya  could we please merge this into the BOAS-430 integration branch?

**Review by mayya-bond-kc — COMMENTED** [review]
Could we create a simple test for this endpoint as well please?

File: apps/api/src/server/applications/application/application.service.js [hunk 1]
```diff
@@ -197,3 +197,21 @@ export const getCaseDetails = async (id, query) => {
...
+export const getFolderDetails = async () => {
+	const folders = [
+		{
+			id: 1,
+			displayNameEn: 'Post-decision',
+			displayOrder: 1100,
+			parentFolderId: null,
+			caseId: null
+		}
+	];
```
[the remark this answers is not in the document]
Comment (Xfarya) [reply apps/api/src/server/applications/application/application.service.js:214]: how would i make the swagger documentation match up to this?

and what is the swagger autogen command?

[the remark this answers is not in the document]
Comment (mayya-bond-kc) [reply apps/api/src/server/applications/application/application.service.js:214]: Swagger is a way for us to document what we expect the endpoint to do, right? As I said in my previous comment, we do not need to return the `parentFolderId` and `caseId` for each folder in the response. Therefore I would create some kind of cleaning mechanism that filters these fields out

File: package-lock.json [hunk 2]
```diff
@@ -3806,6 +3806,30 @@
 				"node": ">=6.0.0"
 			}
 		},
+		"node_modules/@jridgewell/source-map": {
```
[the remark this answers is not in the document]
Comment (Xfarya) [reply package-lock.json:3809]: i did an npm i

[the remark this answers is not in the document]
Comment (mayya-bond-kc) [reply package-lock.json:3809]: Hmm, When I run `npm i` on my local, I don't see these changes in my `package-lock.json`. Is your `package.json` clean?

Also, we generally only run `npm i` when we are adding new packages. If there are no new packages, I would recommend using `npm ci`, which installs packages only in the `package-lock.json` file. `npm ci` is the same command that we use in Azure when we are building the docker containers, so this ensures that we are running the same code locally as well

**Pull request merged**